app/controllers/email_addresses_controller.rb

```diff
@@ -28,7 +28,7 @@ def create
 
     respond_to do |format|
       if @email_address.save
-        format.html { redirect_to @email_address.person, notice: 'Email address was successfully created.' }
+        format.html { redirect_to @email_address.contact, notice: 'Email address was successfully created.' }
```

Comment:
Line is too long. [109/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.